stock_orderpoint_origin/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Open Source Integrators
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Stock Orderpoint Replenishment demand origin details",
+    "summary": "Link Purchase Orders to the replenishment demand Sales Orders",
+    "version": "16.0.1.0.2",
```

Comment:
Whenever we release a new major version, it should start with `1.0.0`.
```suggestion
    "version": "16.0.1.0.0",
```